src/content-cli-import.ts

```diff
@@ -12,10 +12,10 @@ export class Import {
             .description("Command to import all given packages")
             .option("-p, --profile <profile>", "Profile which you want to use to list packages")
             .option("--spaceMappings <spaceMappings...>", "List of mappings for importing packages to different target spaces. Mappings should follow format 'packageKey:targetSpaceKey'")
-            .option("--exportedDatapoolsFile <exportedDatapoolsFile>", "Exported datapool file (relative path)", "")
+            .option("--dataModelMapping <dataModelMapping>", "Data model mappings file (relative path)", "")
```

Comment:
did we not decide we will be sending a mapping like the spaceMapping and not the file?

Reply:
We will recieve the mapping in the file. As opposed to proving the exported datapools as before.